_data/authors.yml

```diff
@@ -42,6 +42,9 @@ lian-yuanlin:
   name: Lian Yuanlin
   thumbnail: /img/authors/lian-yuanlin.jpg
   github: alienchow
+muchen-tang:
+  name: Muchen Tang
+  thumbnail: /img/authors/muchen-tang.jpg
```

Comment:
Try to find/ask for a better profile picture. After the circular border is applied, 1/3 of his face is gone. Ideally his face should be in the center.

Reply:
asked alr, he is fine with it and do have other photos at the moment.